python/src/aiconfig/schema.py

```diff
@@ -664,6 +668,20 @@ def update_prompt(self, prompt_name: str, prompt_data: Prompt):
                 self.prompt_index[prompt_data.name] = prompt_data
                 break
 
+    def _strip_model_names_in_prompt_data(self, prompt_data: Prompt):
+        """
+        Strips model names from the prompt data
+        """
+        if (prompt_data.metadata is not None and prompt_data.metadata.model is not None):
+            model = prompt_data.metadata.model
+            if isinstance(model, str):
+                model = model.strip()
+            else:
+                model.name = model.name.strip()
+                
+                if model.settings is not None and "model" in model.settings:
+                    model.settings["model"] = model.settings["model"].strip()
```

Comment:
This needs to check if it's a string first, right?

Reply:
You're right, thanks for the check